packages/cli/generators/project/templates/.vscode/settings.json

```diff
@@ -4,6 +4,10 @@
   "editor.tabSize": 2,
   "editor.trimAutoWhitespace": true,
   "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": true,
```

Comment:
Also added `editor.codeActionsOnSave.source.organizeImports: true,` as nice to have.